### ISSUE TITLE
Focus not visible for search button in catalog search component

### ISSUE THREAD

**Issue by wlla** [NYPL/design-toolkit#198]
Tested on PC with Chrome/Firefox. Focus is not visible enough around the submit button (`id="nypl-omni-button"`) of the nypl omni search component. Need focus to be visible.

![image](https://user-images.githubusercontent.com/10055967/28926890-2505c50e-7837-11e7-9b3b-4255577379c8.png)

Duplicate of: https://github.com/NYPL-discovery/discovery-front-end/issues/646
